src/main/scala/org/ergoplatform/nodeView/mempool/OrderedTxPool.scala

```diff
@@ -122,14 +122,15 @@ case class OrderedTxPool(orderedTransactions: TreeMap[WeightedTxId, UnconfirmedT
     */
   def canAccept(unconfirmedTx: UnconfirmedTransaction): Boolean = {
     val tx = unconfirmedTx.transaction
-    !isInvalidated(tx.id) && !contains(tx.id) && size <= mempoolCapacity
+    !contains(tx.id) && size <= mempoolCapacity
   }
 
-  def contains(id: ModifierId): Boolean = transactionsRegistry.contains(id)
+  def contains(id: ModifierId): Boolean = {
+    transactionsRegistry.contains(id) || isInvalidated(id)
```

Comment:
`isInvalidated` here is a bit misleading

Reply:
yes, but `contains` used in p2p code to check if transaction is known to mempool. Could be renamed accordingly, but this requires changes in all the ContainsModifiers hierarchy